v5/Http/Resources/PostResource.php

```diff
@@ -123,6 +124,14 @@ private function hydrateResourceRelationships($request)
         return $result;
     }
 
+    private function getPostSource()
+    {
+        $message = $this->message;
+        return $message && isset($message->type)
+        ? $message->type
```

Comment:
@Mh-Asmi last thing would be to fix the indentation

So something like:
```php
return $message && isset($message->type)
    ? $message->type
    : self::DEFAULT_SOURCE_TYPE;
```

Reply:
@Mh-Asmi we would like to push this one. 🙏 Can you apply Kuba's adjustment suggestion, please?